tests/lib/uc20-recovery.sh

```diff
@@ -10,9 +10,18 @@ transition_to_recover_mode(){
         HAVE_LABEL=0
     fi
 
+    # TODO: the following mocking of systemctl should be combined with the code
+    # in tests/core/snapd-refresh-vs-services-reboots into a generic shutdown
+    # helper to get better observability and less race conditions around 
+    # snapd rebooting things from a live system under spread
+
+    # save the original systemctl command since we essentially need to mock it
+    cp /bin/systemctl /tmp/orig-systemctl
+    chmod +x /tmp/orig-systemctl
```

Comment:
from recent discussion of similar code, is this also unneeded?

Reply:
removed